pkg/health/health.go

```diff
@@ -66,18 +67,31 @@ func (s *ServiceNodeStatus) IsCurrentVersion(version string) bool {
 
 type ConsulHealthChecker struct {
 	Health *consulapi.Health
+	Store  kp.Store
 }
 
-func NewConsulHealthChecker(consulHealth *consulapi.Health) *ConsulHealthChecker {
+func NewConsulHealthChecker(store kp.Store, consulHealth *consulapi.Health) *ConsulHealthChecker {
 	return &ConsulHealthChecker{
 		Health: consulHealth,
+		Store:  store,
 	}
 }
 
-func (s *ConsulHealthChecker) toNodeStatus(entry consulapi.ServiceEntry) *ServiceNodeStatus {
+func (s *ConsulHealthChecker) toNodeStatus(serviceID string, entry consulapi.ServiceEntry) *ServiceNodeStatus {
+
+	version := "unknown"
+	manifest, _, _ := s.Store.Pod(kp.RealityPath(entry.Node.Node, serviceID))
```

Comment:
the manifest would be nil in that case, wouldn't it? But I'll check the err as well.